plugins/catalog-react/src/hooks/useEntity.ts

```diff
@@ -51,13 +53,13 @@ export const useEntityFromUrl = (): EntityLoadingStatus => {
     }
   }, [errorApi, navigate, error, loading, entity, name]);
 
-  return { entity, loading, error };
+  return { entity, loading, error, retry };
```

Comment:
`retry` is probably not the right verb we want to use here throughout context. Maybe `refresh` is better? `retry` kind of assumes that the previous call failed.

Reply:
done